src/condor_dagman/condor_submit_dag.cpp

```diff
@@ -33,6 +33,9 @@
 #include "setenv.h"
 #include "condor_attributes.h"
 
+// From condor_utils/condor_config.C
+// Note: This function is declared 'extern "C"'
+bool is_piped_command(const char* filename);
```

Comment:
This comment is a lie, though it was once true.  Can we remove the three lines, and put the declaration of

`bool is_piped_command(const char *filename)` up in condor_config.h or somewhere else globally?